docs/Transformers.md

```diff
@@ -0,0 +1,77 @@
+# Transformers
+
+A transformer allows you to access and perform transformative operations on an asset.
+
+Transformers are defined in: `./packages/configs/default/index.json`
+
+Transformers are ran in order by file type. It is recommended to have more specialized transformers first, and have them skip assets that don't match their case.
```

Comment:
I am not sure how true this is. I believe it is true based on my limited experience, but I am not confident in this.